<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogStudent</class>
 <widget class="QDialog" name="DialogStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить студента</string>
  </property>
  <widget class="QLabel" name="lab_fio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_fio">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_fio_rod">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО (род. под)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_fio_rod">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_orig1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_orig1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>% оригинальности 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_orig2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>% оригинальности 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_orig2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_tema">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тема</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_tema">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_dopusk">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Допуск</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_dopusk">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>?</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lab_comment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Комментарий</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_student">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_comment">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>191</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_nzakl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер заключения</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_nzakl">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_date_zakl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата заключения</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_date_zakl">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
